config/install/block.block.apigee_material_ui_breadcrumbs.yml

```diff
@@ -0,0 +1,22 @@
+uuid: 805f4048-5847-442d-9193-71db26ecaf87
```

Comment:
[Remove uuids from config files](
https://www.drupal.org/docs/8/creating-custom-modules/include-default-configuration-in-your-drupal-8-module)

Reply:
Will do. Thanks @cnovak